src/config/SuffixList.json

```diff
@@ -153,7 +153,7 @@
       {
         "stage": "Excess pre library",
         "suffix": "HXPL",
-        "text": "Excess pre library",
+        "text": "HXPL - Excess pre library",
```

Comment:
Just a doubt - HXPL is added as 'suffix' as well in 'text' as 'HXPL-"? Is it required to add in both places?

Reply:
Yes. The text appears in the page so they can make an informed choice. I did think about joining them together in the page itself but it meant adding more complexity in code so it just seemed better to use the json.